src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -360,10 +377,26 @@ public void reportNoMatchArtefacts() {
     }
 
     /**
-     * Delete expired artefacts from database and Azure storage.
+     * Delete expired artefacts from the database, Artefact and Publications Azure storage.
      */
     public void deleteExpiredArtefacts() {
-        deleteExpiredBlobs(artefactRepository.findOutdatedArtefacts(LocalDate.now()));
+        List<Artefact> outdatedArtefacts = artefactRepository.findOutdatedArtefacts(LocalDateTime.now());
+        outdatedArtefacts.forEach(artefact -> {
+            azureBlobService.deleteBlob(getUuidFromUrl(artefact.getPayload()));
+            // Only attempt to delete from the publications container if it's not a flat file being deleted
+            if (!artefact.getIsFlatFile()) {
+                azureBlobService.deletePublicationBlob(artefact.getArtefactId() + ".pdf");
+                // If it's an SJP list the xlsx file also needs to be deleted
+                if (ListType.SJP_PUBLIC_LIST.equals(artefact.getListType())
+                    || ListType.SJP_PRESS_LIST.equals(artefact.getListType())) {
+                    azureBlobService.deletePublicationBlob(artefact.getArtefactId() + ".xlsx");
+                }
+            }
+        });
+
+        artefactRepository.deleteAll(outdatedArtefacts);
+        log.info("{} outdated artefacts found and deleted for before {}", outdatedArtefacts.size(),
+                 LocalDateTime.now());
```

Comment:
Only a log, but probably worth using the same LocalDateTime as on line 383 so it's consistant.

Also could we update this log to the PIP logger while we're in here

Reply:
Done